apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/metadata.json

```diff
@@ -12,18 +14,20 @@
     "description" : "gov.cms.bfd:bfd-server-war",
     "url" : "https://localhost:IGNORED_FIELD/v2/fhir"
   },
-  "fhirVersion" : "4.0.0",
-  "format" : [ "application/fhir+xml", "application/fhir+json" ],
+  "fhirVersion" : "4.0.1",
+  "format" : [ "application/fhir+xml", "xml", "application/fhir+json", "json" ],
   "rest" : [ {
     "mode" : "server",
     "resource" : [ {
       "type" : "Claim",
       "profile" : "http://hl7.org/fhir/StructureDefinition/Claim",
       "interaction" : [ {
-        "code" : "read"
-      }, {
         "code" : "search-type"
+      }, {
+        "code" : "read"
       } ],
+      "searchInclude" : [ "*", "Claim:mbi" ],
+      "searchRevInclude" : [ "Claim:mbi", "ClaimResponse:mbi", "Coverage:beneficiary", "ExplanationOfBenefit:patient" ],
```

Comment:
This looks like it's providing the search capabilities of the structures that are already in place.  PACA code seems to have mbi as part of the structures for Claim and ClaimResponse as well as the ability to search for a Patient, within the C/CR, by that mbi element.  I think we are good here.

Reply:
fyi @karlmdavis